src/ServiceControl.Transports.ASB/ASBEndpointTopologyTransportCustomization.cs

```diff
@@ -37,12 +39,18 @@ public override void CustomizeRawEndpoint(RawEndpointConfiguration endpointConfi
             transport.UseEndpointOrientedTopology();
             transport.ApplyHacksForNsbRaw();
             ConfigureTransport(transport, transportSettings);
+
+            endpointConfig.LimitMessageProcessingConcurrencyTo(Math.Min(Environment.ProcessorCount, transportSettings.MaxConcurrency));
```

Comment:
This should be MessageReceivers().NumberOfClientsPerEntity(Math.Min(Environment.ProcessorCount, transportSettings.MaxConcurrency))

Reply:
probably want to move it to ConfigureTransport as well